config/software/procps-ng.rb

```diff
@@ -18,9 +18,10 @@
   ship_source "http://dd-accgent-omnibus.s3.amazonaws.com/#{name}-#{version}.tar.xz"
   ship_license "https://gitlab.com/procps-ng/procps/raw/master/COPYING"
   command(["./configure",
-     "--prefix=#{install_dir}/embedded",
-     ""].join(" "),
-  :env => env)
+           "--prefix=#{install_dir}/embedded",
+           "--without-ncurses",
```

Comment:
I think we can remove it from procps-ng as a dependency. However, we cannot remove it from the project. It's a python dependency, along with a few others. (Some of the others we might not actually be using, like erlang, but I know we use python.)